src/preferences/dialog/dlgprefcolors.cpp

```diff
@@ -68,17 +68,15 @@ DlgPrefColors::DlgPrefColors(
             this,
             &DlgPrefColors::slotReplaceCueColorClicked);
 
-    loadSettings();
+    setScrollSafeGuardForAllInputWidgets(this);
```

Comment:
since it is a member function, it already has the "this" pointer.

Reply:
:+1: